index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css"
+      integrity="sha384-b6lVK+yci+bfDmaY1u0zE8YYJt0TZxLEAFyYSLHId4xoVvsrQu3INevFKo+Xir8e"
+      crossorigin="anonymous"
+    />
+    <title>Mobile-First</title>
+  </head>
+  <body>
+    <main>
+    <section class="parent">
+      <div>
+        <nav>
+          <!--container 1-->
+          <div>
+            <h5>My Logo</h5>
+          </div>
```

Comment:
- Kindly wrap your logo inside an `<a>` tag. Your logo will be used to navigate to the home page